src/data_vis.ts

```diff
@@ -174,12 +174,16 @@ class DataVisModel {
             case "fvps":
                 return "fvps"
             case "coverage":
-                return "coverage"
-            case "casesRate":
+                this.cumulativePlot(false); // this might not be prefered
```

Comment:
One way to avoid that behaviour would be to have two properties, `this.cumulativePlotSelectedByUser` which corresponds to the current property, and `this.cumulativePlot` which could be a computed variable:

```
this.cumulativePlot = ko.computed(() => {  
    return ['deaths_averted', 'cases_averted', 'dalys_averted', 'fvps'].includes(this.burdenOutcome()) && this.cumulativePlotSelectedByUser();
})
```
Since the first half of that condition if the same as the condition for the visible binding maybe you want to pull it out into it's own property that you can use in both places:
```
this.canBeCumulative = ko.computed(() => {
   return ['deaths_averted', 'cases_averted', 'dalys_averted', 'fvps'].includes(this.burdenOutcome())
})
```